src/MISC/fix_eco_force.h

```diff
@@ -0,0 +1,91 @@
+/* ----------------------------------------------------------------------
+ LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+ http://lammps.sandia.gov, Sandia National Laboratories
+ Steve Plimpton, sjplimp@sandia.gov
+
+ Copyright (2003) Sandia Corporation.  Under the terms of Contract
+ DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+ certain rights in this software.  This software is distributed under
+ the GNU General Public License.
+
+ See the README file in the top-level LAMMPS directory.
+ ------------------------------------------------------------------------- */
+
+#ifdef FIX_CLASS
+
+FixStyle(eco/force,FixECOForce)
+
+#else
+
+#ifndef LMP_FIX_ECO_FORCE_H
+#define LMP_FIX_ECO_FORCE_H
+
+#include "fix.h"
+
+namespace LAMMPS_NS {
+
+#define FIX_ECO_FORCE_MAX_NEIGHBORS 24
```

Comment:
this define can move with the full declaration of the struct into the .cpp file.